CollectBones.js

```diff
@@ -0,0 +1,93 @@
+// const Matrix = require("./Matrix");
+
+class playerMatrix extends Matrix {
```

Comment:
playerMatrix is not such a proper name for this... better make it "Game", or "Goldrush"